src/vs/workbench/api/common/extHost.api.impl.ts

```diff
@@ -303,7 +303,10 @@ export function createApiFactoryAndRegisterActors(accessor: ServicesAccessor): I
 				return extHostTerminalService.getDefaultShell(false);
 			},
 			get isTelemetryEnabled() {
-				return extHostTelemetry.getTelemetryEnabled();
+				// return extHostTelemetry.getTelemetryEnabled();
+				// return always false to prevent microsoft built-in extension and
+				// third-party like GHPR to send telemetry data
+				return false;
```

Comment:
cc @akosyakov this is queried internally by the microsoft telemetry package here https://github.com/microsoft/vscode-extension-telemetry/blob/188ee72da1741565a7ac80162acb7a08924c6a51/src/common/util.ts#L17-L19

Reply:
Can we set `isTelemetryEnabled` to off always in the extension host process?

Or maybe there is a way to keep it disabled in settings, but enable only in core, i.e. comment out this check https://github.com/felladrin/openvscode-server/blob/5f46c897c3d87fa7169ada4be4b662f58fdc06c6/src/vs/platform/telemetry/common/telemetryService.ts#L110-L113 instead of enabling settings?

Reply:
oh, just realised you did exactly that, so let's go with it for now?